src/views/home/index.tsx

```diff
@@ -31,6 +36,34 @@ export const HomeView = () => {
     };
   }, [marketEmitter, midPriceInUSD, tokenMap]);
 
+  // TODO: 1. is this the right place for this and 2. is it okay to use two useEffects?
+  useEffect(() => {
+    async function populateOAuthToken() {
+      const code = gh.parseOAuthCode() || null;
+
+      if (code) {
+        const oauthToken = await gh.getOAuthToken(code);

```

Comment:
its anti pattern to have oauthToken as state, and initialize here with the same name

Reply:
Yea I figured I was being weird with the token. Would love to manage this better.

Reply:
yea so lets try renaming this const with something like `ghOauthToken` 
then we do `setOAuthToken(ghOauthToken);`
lmk if works

Reply:
i think this is related to https://github.com/cnp0/repomint/pull/19#discussion_r709729674. unless you're recommending changing `token` to `gOauthToken`.

Reply:
> its anti pattern to have oauthToken as state
this is what freaked me out more. i dont wanna anit-pattern :(

Reply:
nah all good :)

Reply:
should I do anything here? i know redux is huge for state management, but idk if thats how we plan to manage tokens. id imagine its some sort of secure session management that would be responsible for holding onto the token.

Reply:
`token` is all ok.
hmm i dont use session management with redux, as i remember it doesnt store data when browser closes, not even when route changes. We can use localStorage / cookies

Reply:
I'm noob at this. Do we want to merge and split the issue? Otherwise I can try to tackle this over the weekend.

Reply:
Yea let's merge and tackle on next ticket

Reply:
Okay resolving conflicts now.